metal/cloud_test.go

```diff
@@ -28,11 +28,6 @@ const (
 	validPlanName   = "Bill by the hour"
 )
 
-var (
-	validCloud *cloud
-	backend    *store.Memory
-)
```

Comment:
Removing these variables was the easiest way that I could enable testing with different configurations instead of a common shared config.